spec/unit/lib/cloud_controller/permissions_spec.rb

```diff
@@ -665,54 +665,37 @@ module VCAP::CloudController
         context 'and user is an admin' do
           it 'returns true' do
             set_current_user(user, { admin: true })
-            expect(permissions.can_write_to_active_space?(space_guid)).to be true
+            expect(permissions.can_write_to_active_space?(space.id)).to be true
           end
         end
 
         context 'and user is admin_read_only' do
           it 'return false' do
             set_current_user_as_admin_read_only
-            expect(permissions.can_write_to_active_space?(space_guid)).to be false
+            expect(permissions.can_write_to_active_space?(space.id)).to be false
           end
         end
 
         context 'and user is global auditor' do
           it 'return false' do
             set_current_user_as_global_auditor
-            expect(permissions.can_write_to_active_space?(space_guid)).to be false
+            expect(permissions.can_write_to_active_space?(space.id)).to be false
           end
         end
 
         context 'and user is not an admin' do
           it 'return false' do
             set_current_user(user)
-            expect(permissions.can_write_to_active_space?(space_guid)).to be false
+            expect(permissions.can_write_to_active_space?(space.id)).to be false
           end
         end
       end
 
       context 'user has valid membership' do
-        it 'returns true for space developer' do
+        it 'returns true' do
```

Comment:
Here you still have this single test instead of separate ones for space developer, space manager, space auditor and org manager.